File: Python/MathematicalPrograms/3DShapes/VolumeOfCuboid.py
ques=input('Do you wish to find the Volume of Cuboid by 2D Method or via 3D Method? Please enter either 2D or 3D')
if ques=='2D':
    print ('OK.')
    ba=int(input('Enter the Base Area of the Cuboid'))
    Height=int(input('Enter a Height measure'))
    v=ba*Height
    print ('Volume of the Cuboid is', v)
elif ques=='3D':
    Length=int(input('Enter a Length measure'))
    Breadth=int(input('Enter a Breadth measure'))
    Height=int(input('Enter a Height measure'))
    v=Length*Breadth*Height
    print ('Volume of the Cuboid is', v)

else:
    exit()

File: C++/MathematicalPrograms/3DShapes/VolumeOfCuboid.cpp
#include <iostream>
#include <string>

using namespace std;

int main() {
	string answer;
	int ba, height;
	double length3D, breadth3D, height3D;
	int volume;
	cout << "Do you wish to find the Volume of Cuboid by 2D Method or via 3D Method? Please enter either 2D or 3D";
	getline(cin, answer);
	if (answer == "2D") {
		cout << "OK.";
		cout << "\nEnter the Base Area of the Cube";
		cin >> ba;
		cout << "Enter a Height measure";
		cin >> height;
		volume = ba*height;
		cout << "Volume of the Cuboid is " << volume;
	}
	else if (answer == "3D") {
		cout << "OK.";
	    cout << "\nEnter a Length measure";
		cin >> length3D;
		cout << "Enter a Breadth measure";
		cin >> breadth3D;
		cout << "Enter a Height measure";
		cin >> height3D;
		volume = length3D*breadth3D*height3D;
		cout << "Volume of the Cuboid is " << volume;
	}
	else {
		exit(0);
	}
}